pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -274,6 +300,38 @@ func ingest(ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.
 	return errors.CombineErrors(ingestWithClient(), client.Close(ctx))
 }
 
+func ingestWithRetries(
+	ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.Job,
+) error {
+	ro := retry.Options{
+		InitialBackoff: 3 * time.Second,
+		Multiplier:     2,
+		MaxBackoff:     1 * time.Minute,
+		MaxRetries:     60,
+	}
```

Comment:
Steven's [earlier comment](https://github.com/cockroachdb/cockroach/pull/85432#discussion_r942655873) seems like it'd still apply here, where this is expected to be an _incredibly_ long running job and persisting consequences of previous eventually-successful retries is probably not great.

If we you'd rather address that in a different issue/pr I'd at least put a comment here highlighting the concern.